README.md

```diff
@@ -8,129 +10,203 @@
-* [SRT Protocol Technical Overview](https://github.com/Haivision/srt/files/2489142/SRT_Protocol_TechnicalOverview_DRAFT_2018-10-17.pdf)
-* [Why SRT Was Created](docs/misc/why-srt-was-created.md)
+## Features
 
-## Requirements
+> :point_down: Click on the &#9658; button to expand a feature description.
 
-* C++03 (or above) compliant compiler.
-* CMake 2.8.12 or above (as build system).
-* OpenSSL 1.1 (to enable encryption, or build with `-DENABLE_ENCRYPTION=OFF`).
-* Multithreading is provided by either of the following:
-  * C++11: standard library (`std` by `-DENABLE_STDCXX_SYNC=ON` CMake option);
-  * C++03: Pthreads (for POSIX systems it's built in, for Windows there is a ported library).
-* Tcl 8.5 (optional, used by `./configure` script or use CMake directly).
+<details>
+  <summary>Pristine Quality and Reliability</summary>
+
+  <p>
+
+  No matter how unreliable your network, SRT can recover from severe packet loss and jitter, ensuring the integrity and quality of your video streams.
+
+  </p>
+</details>
+
+<details>
+  <summary>Low Latency</summary>
+
+  <p>
+
+  SRT’s stream error correction is configurable to accommodate a user’s deployment conditions. Leveraging real-time IP communications development to extend traditional network error recovery practices, SRT delivers media with significantly lower latency than TCP/IP, while offering the speed of UDP transmission with greatly improved reliability.
+
+  </p>
+</details>
+
+<details>
+  <summary>Content Agnostic</summary>
+
+  <p>
+
+  Unlike some other streaming protocols that only support specific video and audio formats, SRT is payload agnostic. Because SRT operates at the network transport level, acting as a wrapper around your content, it can transport any type of video format, codec, resolution, or frame rate.
+
+  </p>
+</details>
+
+<details>
+  <summary>Easy Firewall Traversal with Rendezvous Mode</summary>
+
+  <p>
+
+  The handshaking process used by SRT supports outbound connections without the potential risks and dangers of permanent exterior ports being opened in a firewall, thereby maintaining corporate LAN security policies and minimizing the need for IT intervention.
+
+  </p>
+</details>
+
+<details>
+  <summary><a href="https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-01#section-6">AES Encryption</a></summary>
+
+  <p>
+
+  Using 128/192/256-bit AES encryption trusted by governments and organizations around the world, SRT ensures that valuable content is protected end-to-end from contribution to distribution so that no unauthorized parties can listen.
+
+  </p>
+</details>
+
+<details>
+  <summary><a href="./docs/features/packet-filtering-and-fec.md">Forward Error Correction (FEC) and Packet Filter API</a></summary>
 
-For a detailed description of the build system and options, please refer to [SRT Build Options](docs/build/build-options.md).
+  <p>
```

Comment:
@ethouris It was written in HTML to take advantage of the collapsible <details> tag. Since the README is the "front face" of the repo, my opinion is that we should be less restrictive about the use of MarkDown in order to have the freedom to format the output to improve its appearance on the main page.